### DISCUSSION TITLE
get snapd to work with blackarch

### DISCUSSION THREAD

**Discussion by maximilien1990** [BlackArch/blackarch#4094] [Q&A] [answered]
- **snapd**:
- **www.snapcraft.io**:
- **to install snap packages**:

i tried making it work and when on the snapcraft forum to ask why it wasnt working and heres what they told me 

Thanks, that explains why things don’t work. Snapd will not be able to identify this distribution as Arch and things will not work out of the box. I think that you should file a bug for Black Arch maintainers and ask them to fill os-release with meaningful information. Some reference is provided in the man pages, eg. https://man.archlinux.org/man/os-release.5 To be at least marginally useful to any software which may consult /etc/os-release, ID or ID_LIKE are the least that’s needed.

**Comment by noraj** [comment, marked as answer]
I don't think there is anyting specific from BA that will impact snap. I think it's AL related, have you read https://wiki.archlinux.org/title/Snap?

**Comment by ikstream** [comment]
Hi @maximilien1990 , can you provide the link to the issue you opened with them
